src/main/java/uk/ac/ebi/biostudies/controller/File.java

```diff
@@ -48,4 +51,28 @@ public ResponseEntity<String> search(@PathVariable(value="accession") String acc
         }
     }
 
+    @RequestMapping(value = "/links/{accession:.+}", produces = JSON_UNICODE_MEDIA_TYPE, method = {RequestMethod.POST, RequestMethod.GET})
```

Comment:
If these are just links inside the files and not all links, the endpoint shoudl reflect that.

Reply:
mining-links?
filelinks ?

Reply:
`extractedlinks`

Reply:
Also, it might make sense to flip the parameters for these calls, for example, `/files/{accession:.+}` should perhaps be `{accession:.+}/files` . Let's add this as a separate ticket

Reply:
For flipping files url, I will do this in another ticket and will keep the current one related to extracted links.
I will apply other changes